release-notes/aws/v9.2.3.md

```diff
@@ -0,0 +1,20 @@
+## :zap: Giant Swarm Release 9.2.3 for AWS :zap:
+
+**If you are upgrading from 9.2.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release includes reliability improvements for NGINX Ingress Controller, most importantly termination configuration was adjusted so that active connections get drained gracefully.
```

Comment:
```suggestion
This release improves the reliability of NGINX Ingress Controller. Most important, termination configuration was adjusted so that active connections now get drained gracefully.
```

Reply:
I added a suggestion to imply the change from previous to before.